test/e2e/testdata/artifactgc/artgc-template.yaml

```diff
@@ -6,9 +6,9 @@ spec:
   workflowMetadata:
     labels:
       workflows.argoproj.io/test: "true"
-  entrypoint: artgc-template
+  entrypoint: artgc-template-main
   templates:
-  - name: artgc-template
+  - name: artgc-template-main
     container:
```

Comment:
wanted different names for WorkflowTemplate and template, just in case a test were to falsely pass due to the name being the same.